### PR TITLE
Adding example.py to my_branch_eleusis_b1

### PR DESCRIPTION
**Pull request by Eleusis1995**
Adding example.py: sorting and printing numeric list

### CODE REVIEW

File: example.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+l_my = [1,2,56,21,32,9]
```
Comment (AlexNoelHdz) [inline example.py:1]: Please make this list more descriptive. f.e. number_list

File: example.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+l_my.sort()
+for l in l_my:
```
Comment (AlexNoelHdz) [inline example.py:3]: you can use "number" instead of just "l"

**Review by AlexNoelHdz — CHANGES_REQUESTED** [review]

File: example.py (hunk 1 above)
Comment (Eleusis1995) [reply example.py]: Ok, l_my changeed its name to numbers_unsorted

File: example.py (hunk 2 above)
Comment (Eleusis1995) [reply example.py]: l changed to number

**Review by Eleusis1995 — COMMENTED** [review]
The sugestions were made

**Review by AlexNoelHdz — APPROVED** [review]

**Pull request merged**